src/new_circuit.jl

```diff
@@ -0,0 +1,8 @@
+function new_circuit(feature_map_circuit::AbstractBlock, var::AbstractBlock, x, theta, N::Int, n=1, v=0; mapping::FeatureMap)
+	f = repeat([phi(x, mapping)], N)
+	ignore() do
+		f[n] = f[n] + v
+	end
```

Comment:
that ignore will make this not differentiate correctly. Is that the purpose'?

Reply:
That is on purpose. It isn't affecting the program from what I've seen. 
@ChrisRackauckas I'll change it to `Buffer`, if I encounter a problem with this or Buffers are faster in general.